packages/xstate-graph/src/index.ts

```diff
@@ -1,15 +1,24 @@
 export {
   getStateNodes,
-  getPathFromEvents,
-  getSimplePlans,
-  getShortestPlans,
   serializeEvent,
   serializeMachineState as serializeState,
   toDirectedGraph,
-  performDepthFirstTraversal,
-  traverseShortestPlans,
-  traverseSimplePlans,
-  traverseSimplePathsTo
+  joinPaths,
+  AdjacencyMap,
+  AdjacencyValue
 } from './graph';
+export {
+  getMachineSimplePaths,
+  getSimplePaths,
+  getSimplePathsFromTo
+} from './simplePaths';
+export {
+  getShortestPaths,
+  getMachineShortestPaths,
+  getShortestPathsFromTo,
+  getMachineShortestPathsFromTo
+} from './shortestPaths';
```

Comment:
The balance is missing here :P It seems that there is no `getMachineSimplePathsFromTo`

Reply:
215bbbeb0572c8c4d178605ad4cf1be5c953b009